notus/scanner/loader/json.py

```diff
@@ -108,7 +116,7 @@ def load_package_advisories(
                     else:
                         package = RPMPackage.from_full_name(full_name)
                 else:
-                    if "debian" in operating_system:
+                    if "debian" in operating_system or "debian" in package_type:
```

Comment:
When looking at the spec the package_type should be `deb` in this case. Additionally I suppose we should also use `operating_system.lower()` here.

Reply:
you're right it is deb; I think it is better to just use package_type and validate setting and drop the operating system check completely here; also switching to case class and  enum to make it a bit more explicit; this commit was more of a now it is at least at a testable state. btw you noticed that it is a draft pr? so you could just change things :)

Reply:
```suggestion
                    if package_type == "deb":
```
I would rather just change it to this and map all supported package types.

Reply:
maybe not case class for the whole file since it will name clash a lot; just a enum :D